_posts/2025-11-17-documenting-package-manager-data.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: "Documenting Package Manager Data"
+date: 2025-11-17 12:00 +0000
+---
+
+Recently we've been working with the [CHAOSS Package Metadata Working Group](https://github.com/chaoss/wg-package-metadata) and [Alpha-Omega](https://alpha-omega.dev) to better document the similarities and differences in the variety of package manager clients and registries.
+
+Information about package managers is spread out across different sources, formats, and documentation styles, which can be a blocker when it comes to learning and contributing. Last week we published five repositories documenting different aspects of how package managers work.
```

Comment:
Feel like this might be the better opener i.e. 'this is the problem' and then 'this is what we've been doing about it. 

Also is there a purpose beyond 'learning and contributing'. Is the overall goal to produce a set of recommendations and suggested behaviours (I dont want to say 'best practices') for package managers?